ModTek/Logging/BetterLogHandler.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using HBS.Logging;
+using Logger = HBS.Logging.Logger;
+using Object = UnityEngine.Object;
+
+namespace ModTek.Logging
+{
+    internal class BetterLogHandler
+    {
+        internal static BetterLogHandler Shared = new BetterLogHandler();
+
+        private ILogAppender cleanAppender;
+        private LogLevel logLevel;
+        //private bool ignoreLoggerLogLevel = true; // logger log level is hidden
+
+        internal void SetupCleanedLog(string path, BetterLogSettings settings, bool enableStackTraceLogging)
+        {
+            cleanAppender = new BetterLogFileAppender(path, settings.Formatter, settings.PatternsToIgnore);
+            logLevel = settings.LogLevel;
+            Logger.IsStackTraceEnabled = enableStackTraceLogging;
+        }
+
+        internal void SetupModLog(ModDef modDef)
+        {
+            var settings = modDef.LogSettings;
+            var loggerName = modDef.Name;
+            var logPath = Path.Combine(modDef.Directory, "log.txt");
```

Comment:
the location of the log file should be fixed, so if someone tells me to get the log files of a mod, I expect it to be the standard name at the standard location